modules/ROOT/partials/release-notes/6.0-release-notes-ui-and-ux-elements-and-components.adoc

```diff
@@ -54,9 +54,14 @@ Consequently, it now supports partial nested objects.
 [[editor-text-entry-height]]
 === Editor text entry height
 
-In {productname} 5.x, editors without a specified `height` value defaulted to 200px high.
+In {productname} 5.x, editors without a `height` value specified in the {productname} configuration defaulted to a 200px high text entry area.
```

Comment:
This change isn't quite correct... the 200px was the entire editor, not just the text entry area (e.g. it included the menu bar and toolbar).

Reply:
Corrected.